make setup fails

```
~/workspace/oml > make setup
opam install lacaml lbfgs ocephes kaputt dsfo bisect_ppx
[ERROR] No package named dsfo found.
make: *** [setup] Error 66
```

opam pinning https://github.com/rleonid/dsfo worked to get around this (though, horribly, reinstalled every single ocaml library on my computer; thank you aspcud?)

--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,7 @@ all: build
 
 # This should be called something else.
 setup:
+	opam pin add dsfo git://github.com/rleonid/dsfo
 	opam install $(PACKAGES_INSTALL)
 
 oml.cmxa:

